bitwarden_license/src/app/organizations/manage/sso.component.ts

```diff
@@ -117,45 +118,56 @@ export class SsoComponent implements OnInit {
     }
 
     async submit() {
+        this.formPromise = this.postData();
+
+        let response: OrganizationSsoResponse;
+        try {
+            response = await this.formPromise;
+        } catch (e) {
+            // Logged by apiService, do nothing
+            return;
+        }
```

Comment:
If we're just going to catch and do nothing, what's the benefit vs letting the error propagate?

Reply:
Good point, I moved this into a `finally` block